Makefile

```diff
@@ -131,38 +132,40 @@ build-itest-race:
 	@$(call print, "Building itest binary for ${backend} backend.")
 	CGO_ENABLED=0 $(GOTEST) -v ./itest -tags="$(DEV_TAGS) $(RPC_TAGS) integration $(backend)" -c -o itest/itest.test$(EXEC_SUFFIX)
 
-#? install-binaries: Build and install lnd and lncli binaries, place them in $GOPATH/bin
+#? install-binaries: Build and install lnd, lncli and lndsigner binaries, place them in $GOPATH/bin
 install-binaries:
-	@$(call print, "Installing lnd and lncli.")
+	@$(call print, "Installing lnd/lndsigner and lncli.")
 	$(GOINSTALL) -tags="${tags}" -ldflags="$(RELEASE_LDFLAGS)" $(PKG)/cmd/lnd
 	$(GOINSTALL) -tags="${tags}" -ldflags="$(RELEASE_LDFLAGS)" $(PKG)/cmd/lncli
+	$(GOINSTALL) -tags="${LND_SIGNER_TAGS}" -ldflags="$(RELEASE_LDFLAGS)" $(PKG)/cmd/lndsigner
 
 #? manpages: generate and install man pages
 manpages:
 	@$(call print, "Generating man pages lncli.1 and lnd.1.")
 	./scripts/gen_man_pages.sh $(DESTDIR) $(PREFIX)
 
-#? install: Build and install lnd and lncli binaries and place them in $GOPATH/bin.
+#? install: Build and install lnd, lncli and lndsigner binaries and place them in $GOPATH/bin.
 install: install-binaries
 
 #? install-all: Performs all the same tasks as the install command along with generating and
 # installing the man pages for the lnd and lncli binaries. This command is useful in an
 # environment where a user has root access and so has write access to the man page directory.
 install-all: install manpages
 
-#? release-install: Build and install lnd and lncli release binaries, place them in $GOPATH/bin
+#? release-install: Build and install lnd, lncli and lndsigner binaries release binaries, place them in $GOPATH/bin
 release-install:
-	@$(call print, "Installing release lnd and lncli.")
+	@$(call print, "Installing release lnd/lndsigner and lncli.")
 	env CGO_ENABLED=0 $(GOINSTALL) -v -trimpath -ldflags="$(RELEASE_LDFLAGS)" -tags="$(RELEASE_TAGS)" $(PKG)/cmd/lnd
 	env CGO_ENABLED=0 $(GOINSTALL) -v -trimpath -ldflags="$(RELEASE_LDFLAGS)" -tags="$(RELEASE_TAGS)" $(PKG)/cmd/lncli
```

Comment:
Something to note here is that `lncli` will be built with the `RELEASE_TAGS` build tags, and not with the `LND_SIGNER_TAGS` that the actual `lncli` that's shipped along with `lndsigner` in the zip file. I.e. the `lncli` here will contain more functionality. IMO that's ok though, as it could be a bit confusing if we'd allow installing of 2 different `lncli` in this `make` command.
Potentially we could give an option to the executer of this command choose which version they'd like to install though.